### PR TITLE
add props maxLength, autofocus to EditableTitle

### PR DESCRIPTION
**Pull request by adirou**
### 🔦 Summary
added autoFocus prop 
added maxLength prop

...

### ✅ Checklist
- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

File: src/EditableTitle/EditableTitle.js [hunk 1]
```diff
@@ -31,7 +37,7 @@ class EditableTitle extends React.Component {
     });
   };
   render() {
-    const { dataHook, className } = this.props;
+    const { dataHook, className, maxLength } = this.props;
```
Comment (shlomitc) [inline src/EditableTitle/EditableTitle.js]: add to proptypes and add missing test for both custom and default values

Comment (adirou) [reply src/EditableTitle/EditableTitle.js]: there are not tests in the input wsr for length , besides how do you want us to test it , the inputDriver doesn't have getProps method

Comment (adirou) [reply src/EditableTitle/EditableTitle.js]: i can test the behaviour of WSR input but it's doesn't make sense

Comment (shlomitc) [reply src/EditableTitle/EditableTitle.js]: so maybe it's a good time to do it. you can add to the input test and driver `getMaxLength: () => input.maxLength` and use it also in the EditableTitle.

Comment (shlomitc) [reply src/EditableTitle/EditableTitle.js]: also, the autoFocus should be tested as well

Comment (adirou) [reply src/EditableTitle/EditableTitle.js]: updated

**Pull request merged**